pkg/controller/jaeger/elasticsearch.go

```diff
@@ -83,6 +83,21 @@ func waitForAvailableElastic(c client.Client, es esv1.Elasticsearch) error {
 				available++
 			}
 		}
+		ssList := corev1.StatefulSetList{}
```

Comment:
What's the relationship between the deployment above and this stateful set? Is it sufficient to just check the stateful set?

Reply:
SS is used exclusively for _only_ master nodes. The deployments are used for data or other combinations.